packages/spotlight/src/spotlight.js

```diff
@@ -354,6 +355,12 @@ const Spotlight = (function () {
 		_pointerMoveDuringKeyPress = false;
 	}
 
+	function handleWebOSMouseEvent (ev) {
+		if (ev && ev.detail && ev.detail.type === 'Leave') {
```

Comment:
Do we need to detect an 'Enter' event to reset the pointer mode?  Is it possible for pointer mode to change after the 'leave' and still get an 'enter' event?  I don't know!

Reply:
no we don't need the enter event handler. We tested.